pkg/dt/fdt.go

```diff
@@ -411,3 +413,35 @@ func (fdt *FDT) Write(f io.Writer) (int, error) {
 	_, err := w.Write(strs)
 	return w.N, err
 }
+
+// NodeByName finds a node by name.
+func (fdt *FDT) NodeByName(name string) (*Node, bool) {
+	return fdt.RootNode.Find(func(n *Node) bool {
+		return n.Name == name
+	})
+}
+
+// Bytes returns a []byte for the "data" property of a node,
+// or an error if the node has no data property.
+func (fdt *FDT) Bytes(name string) ([]byte, error) {
+	n, ok := fdt.NodeByName(name)
+	log.Printf("Bytes: nodebyname for %q is %s", name, fdt)
+	if !ok {
+		return nil, os.ErrExist
```

Comment:
Can you instead do `fmt.Errorf("could not find device tree node %q", name)`

Reply:
done